<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1489</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертер статей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
        <item>
         <widget class="QLabel" name="hashStatusLabel">
          <property name="text">
           <string>Чтобы проверить хэш статьи - нажмите кнопку &quot;Конвертировать&quot;</string>

          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="inFileText">
          <property name="placeholderText">
           <string>Введите сюда исходный текст</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="convertBtn">
          <property name="text">
           <string>Конвертировать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item>
         <widget class="QComboBox" name="outFileFormatBox">
          <property name="placeholderText">
           <string>Формат выходного файла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="outFileText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Здесь Вы увидите результат конвертации</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1489</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
